lib/timeline/timeline.dart

```diff
@@ -290,6 +292,37 @@ class TimelineScreen extends Screen {
 
     return button;
   }
+
+  // TODO: move this button out of timeline if we decide to make a global button bar.
```

Comment:
this button seems pretty similar to the hot reload one. Consider making a base class for buttons calling a service extension of this form. I expect this isn't the last one we will add.

Reply:
Done.